src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVIntervalTree.java

```diff
@@ -0,0 +1,904 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
```

Comment:
@tedsharpe I just want to check that you know that there is already an interval tree in htsjdk that you implemented that seems to have very similar functionality.